package.json

```diff
@@ -45,16 +45,15 @@
   "dependencies": {
     "@babel/eslint-plugin": "^7.19.1",
     "@babel/plugin-transform-modules-commonjs": "^7.21.2",
-    "@enact/dev-utils": "^5.1.2",
-    "@enact/template-sandstone": "^1.5.2",
+    "@enact/dev-utils": "enactjs/dev-utils#feature/WRP-11567",
```

Comment:
After releasing `dev-utils v5.1.3`, I will change this line to `^5.1.3`

Reply:
I revert this change.
dev-utils v5.1.3 does not have to be included and there is no problem with the actual operation.
It will be updated again in the CLI v6.0.0 release.